lib/ui/discover/discover_child_item.dart

```diff
@@ -7,81 +7,58 @@ import 'package:farmsmart_flutter/utils/styles.dart';
 import 'package:flutter/material.dart';
 import 'package:farmsmart_flutter/utils/assets.dart';
 
+import 'discover_page.dart';
+
 class DiscoveryListItem {
-  Widget buildListItem(ArticleEntity articleData, Function getRelatedArticles) {
+  Widget buildListItem(ArticleEntity articleData, Function getRelatedArticles, HomeDiscoverPageStyle discoverStyle) {
     return GestureDetector(
-        onTap: () {
-          getRelatedArticles(articleData);
-        },
-        child: Padding(
-          key: ValueKey(articleData.title ?? Strings.noTitleString),
-          padding: Paddings.listOfArticlesPadding(),
-          child: Container(
-              height: 101,
-              child: Column(
+      onTap: () => getRelatedArticles(articleData),
+      child: Container(
+        child: Column(
+          children: <Widget>[
+            Container(
+              padding: discoverStyle.itemListPadding,
+              child: Row(
+                mainAxisAlignment: discoverStyle.centerAlignmentVertical,
                 children: <Widget>[
-                  Row(
-                    children: <Widget>[
-                      _buildListItemImage(articleData),
-                      Padding(
-                        padding: Paddings.leftPaddingSmall(),
-                      ),
-                      _buildArticleTitle(articleData),
-                      _buildListIcon(),
-                      Padding(
-                        padding: Paddings.rightPaddingSmall(),
-                      ),
-                    ],
-                  ),
-                  Padding(
-                    padding: Paddings.bottomPaddingSmall(),
-                  ),
-                  Dividers.listDividerLine()
+                  _buildArticleInformation(articleData, discoverStyle),
+                  SizedBox(width: discoverStyle.spaceBetweenItemTextAndImage),
+                  _buildListItemImage(articleData, discoverStyle),
                 ],
-              )),
-        ));
+              ),
+            ),
+            buildListSeparator(discoverStyle)
+          ],
+        ),
+      ),
+    );
   }
 
-  _buildListItemImage(ArticleEntity articleData) {
-    return NetworkImageFromFuture(
-      articleData.imageUrl,
-        height: listImageHeight,
-        width: listImageWidth,
-        fit: BoxFit.cover
+  _buildListItemImage(ArticleEntity articleData, HomeDiscoverPageStyle discoverStyle) {
+    return ClipRRect(
+        borderRadius: BorderRadius.circular(discoverStyle.itemImageBorderRadius),
+        child: NetworkImageFromFuture(
+            articleData.imageUrl,
+            height: discoverStyle.itemImageSize,
+            width: discoverStyle.itemImageSize,
+            fit: BoxFit.cover)
     );
   }
 
-  _buildArticleTitle(ArticleEntity articleData) {
+  _buildArticleInformation(ArticleEntity articleData, HomeDiscoverPageStyle discoverStyle) {
```

Comment:
good style injection, swap out the model object for a viewModel and were looking good 👍